.vscode/tasks.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
should this get readded?

Reply:
It should. I submitted a PR to update our internship master branch with `tasks.json` and waiting for it to be approved.